### DISCUSSION TITLE
Any need for `hibernate-enhance-maven-plugin`?

### DISCUSSION THREAD

**Discussion by agreedSkiing** [quarkusio/quarkus#31463] [Q&A] [answered]
Been looking around and trying to optimize some of our database calls and stumbled upon `org.hibernate.orm.tooling:hibernate-enhance-maven-plugin` but would that be of any use in quarkus? 

I know from a previous question that there already are some bitcode enchaments implemented for the `quarkus-hibernate-orm` depedency

[a comment by an automated account is not shown]

**Comment by Sanne** [comment, marked as answer]
Hi @agreedSkiing ,
no there's no use of applying the Maven plugin to a Quarkus application: the same transformations are applied automatically by the Quarkus extension.
